db/models/aggregate_functions.py

```diff
@@ -214,24 +214,26 @@ def update_application_and_related_form(
 
     application.last_edited = func.now()
     form_sql_row = FormsMethods.get_form(application_id, form_name)
-    # updating project name
-    if form_name == "project-information":
+    update_project_name(form_name, question_json, application)
+    form_sql_row.json = question_json
+    update_statuses(application_id, form_name, is_summary_page_submit)
+    db.session.commit()
+    current_app.logger.info(
+        f"Application updated for application_id: '{application_id}."
+    )
+
+
+def update_project_name(form_name, question_json, application):
```

Comment:
Can we move this up so it's defined before it is referenced?

Reply:
have moved this up 👍🏾